### DISCUSSION TITLE
Problema al realizar git push

### DISCUSSION THREAD

**Discussion by edu18RR** [community/community#31830] [Other Feature Feedback, Questions, & Ideas] [answered]
Hola comunidad,

Estoy tratanto de realizar la sincronización de mi proyecto, la verdad voy iniciando pero la verdad ya no se que hacer .

1. 
![image](https://user-images.githubusercontent.com/81430898/188262768-83950ab1-dce2-4ea3-94c5-374f9d881118.png)

y tambien me podrian ayudar a saber porque no se crea el .pub  al generar el SSH

gracias por su apoyo

**Comment by edu18RR** [comment]
Para la creación de  la llave ssh, se crea un archivo el cual lo idendifica como publisher en W11, solo realice la ocultación de la extención y me permitio, solo quedaria de su apoyo con el error de la imagen.

**Reply by edu18RR** [reply]
Listo encotre un hilo donde destaca lo mismo que me pago si alguien más le sucede les dejo el link

https://github.com/orgs/community/discussions/22075#discussion-4248174

y para lo de SHH les comparto el link

https://github.com/orgs/community/discussions/27738#discussion-4265481

**Comment by melvincwng** [comment, marked as answer]
Hi @edu18RR, try the following command below:

> git pull origin main

This should resolve the issue you are facing. The reason why your git push command failed is because your local branch/repository is behind the remote repository in terms of commits. You need to sync them together using the command above so that subsequent git push commands will work.

Hope this clarifies and answers your question.

**Reply by melvincwng** [reply]
**Translated from English to Spanish:**

Hola @edu18RR , prueba el siguiente comando a continuación:

> git pull origin main

Esto debería resolver el problema al que se enfrenta. La razón por la que tu comando git push falló es porque tu sucursal/repositorio local está detrás del repositorio remoto en términos de confirmaciones. Debe sincronizarlos usando el comando anterior para que los comandos posteriores de git push funcionen.

Espero que esto aclare y responda tu pregunta.

**Reply by melvincwng** [reply]
Si esta respuesta le ha ayudado, por favor, "Marcar como respuesta" para que otros sepan que este problema se ha solucionado.

Gracias, y que tengas un buen día por delante 😄

**Comment by camila-valderrama** [comment]
What can I do to push in Git to a coworker's repository in Github? They ask me to write my Github's username and password but the access fail everytime. I don't know why they ask me that, I have worked before without problems
